build_defs/build_defs.bzl

```diff
@@ -112,7 +118,13 @@ def arcs_ts_test(name, src, deps):
     run_in_repo_test(
         name = name,
         srcs = [src],
-        cmd = "./tools/sigh test --bazel --file {SRC}",
+        cmd = "$(location //tools:sigh_bin) test --bazel --file {SRC}",
         tags = EXECUTION_REQUIREMENTS_TAGS,
-        deps = deps + ["//src:core_srcs"],
+        deps = deps + [
+            "//:all_srcs",
+            "//tools:sigh_bin",
+            "//tools:tools_srcs",
+            "//:node_modules",
+            "//:tsconfig"
```

Comment:
There's a lot of duplicated deps here just needed to run sigh correctly. Adding a separate rule in tools/ to run sigh subcommands might be a good idea. e.g. 

```
sigh_command(
    name = name,
    command = "test --bazel --file {SRC}",
    ...
)
```